lib/logger.js

```diff
@@ -180,7 +180,9 @@ Logger.prototype.log = function(level, var_args) {
  * @param {Array} args The items to log.
  */
 Logger.prototype._log = function(level, args) {
-  var logRecord = new LogRecord(level, this.name, this.info_, args);
+  // Clone to prevent modifications to this Logger's internal info_ through logRecord.meta
+  var clonedInfo_ = Object.assign({}, this.info_)
```

Comment:
instead of cloning, maybe we should Object.create(this.info_)?

that will put it on the prototype chain so that the meta info in the Logger will get inherited